_studio/shared/include/mfx_utils.h

```diff
@@ -131,4 +131,7 @@ template<class T> inline T AlignValue(T value, mfxU32 alignment)
 //#undef  FAILED
 //#define FAILED(hr)      (MFX_STS_TRACE(hr) < 0)
 
+#define mfxCopyField(Field)      buf_dst.Field = buf_src.Field
```

Comment:
Macros should not mimic to ordinary function. Please re-name it.

Reply:
@dmitry-gurulev your suggestion on name changing? caps lock?